backend/src/main/java/com/shipping/backend/entities/QueueRequestMessage.java

```diff
@@ -0,0 +1,14 @@
+package com.shipping.backend.entities;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.io.Serializable;
+
+@Getter  @Setter  @NoArgsConstructor
```

Comment:
You can shrink this by using lombok's @Data. Its designed to be THE lombok annotation for POJOs.

Reply:
Already done will commit and push in the night